packages/core/cache/src/LMDBCache.js

```diff
@@ -53,16 +57,28 @@ export class LMDBCache implements Cache {
   }
 
   async set(key: string, value: mixed): Promise<void> {
-    await this.store.put(key, serialize(value));
+    await this.setBlob(key, serialize(value));
   }
 
   getStream(key: string): Readable {
-    return blobToStream(this.store.get(key));
+    let filename = path.join(this.dir, key);
+    if (this.fs.existsSync(filename)) {
+      return this.fs.createReadStream(filename);
+    } else {
+      // If the file doesn't exists, return an empty stream.
```

Comment:
Hmm, should it error?

Reply:
Yeah, I suppose it should, as that seems to be what FSCache does.